client/src/components/graph/graph.tsx

```diff
@@ -542,7 +549,17 @@ class Graph extends React.Component<GraphProps, GraphState> {
   };
 
   getViewportDimensions = (): { height: number; width: number } => {
-    const { viewportRef } = this.props;
+    const { viewportRef, screenCap } = this.props;
+
+    if (screenCap) {
+      const prevAspectRatio =
+        viewportRef.clientHeight / viewportRef.clientWidth;
+
+      return {
+        height: 1800 * prevAspectRatio,
```

Comment:
I forgot why we picked 1800 instead of any other numbers 😆 

Maybe extract as constant and add comment?

Reply:
lol i miss typed 1080, are we cool with this magic number?